GitUI/Editor/FileViewer.cs

```diff
@@ -600,7 +600,9 @@ private string GetFileText(string fileName)
 
             if (File.Exists(path))
             {
-                using (var reader = new StreamReader(path, Module.FilesEncoding))
+                // StreamReader disposes of 'fileStream'.
+                var fileStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Comment:
I normally change it in my fork, reset, squash, push.
I think I've seen "live edit" feature, though don't remember whether it was in github.com or enterprise edition...